packages/api/src/command/medical/admin/update-hie-enabled-feature-flags.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  getFeatureFlags,
+  createAndDeployHieConfigurationContent,
+  CxFeatureFlagStatus,
+} from "@metriport/core/external/aws/app-config";
+import { Config } from "../../../shared/config";
+
+export async function updateHieEnabledFFs({
+  cxId,
+  cwEnabled,
+  cqEnabled,
+}: {
+  cxId: string;
+  cwEnabled?: boolean;
+  cqEnabled?: boolean;
+}): Promise<CxFeatureFlagStatus> {
+  const region = Config.getAWSRegion();
+  const appId = Config.getAppConfigAppId();
+  const configId = Config.getAppConfigConfigId();
+  const envName = Config.getEnvType();
+  const envId = Config.getAppConfigEnvironmentId();
+  const deploymentStratId = Config.getAppConfigDeploymentStrategyId();
+  const featureFlags = await getFeatureFlags(region, appId, configId, envName);
```

Comment:
ditto: being more specific with what we import

Reply:
See above.